README.md

```diff
@@ -147,40 +156,76 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
      http://scanner-adapter:8080/api/v1/scan/3fa85f64-5717-4562-b3fc-2c963f66afa6/report
+    Content-Type: application/vnd.security.vulnerability.report; version=1.1;
+    Status: 200 OK
 
-   Content-Type: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0
-   Status: 200 OK
-
-   {
-     "generated_at": "2019-08-07T12:17:21.854Z",
-     "artifact": {
-       "repository": "library/mongo",
-       "digest": "sha256:917f5b7f4bef1b35ee90f03033f33a81002511c1e0767fd44276d4bd9cd2fa8e"
-     },
-     "scanner": {
-       "name": "Microscanner",
-       "vendor": "Aqua Security",
-       "version": "3.0.5",
-     },
-     "severity": "High",
-     "vulnerabilities": [
-       {
-         "id": "CVE-2017-8283",
-         "package": "dpkg",
-         "version": "1.17.27",
-         "fix_version": "1.18.0",
-         "severity": "High",
-         "description": "...",
-         "links": [
-           "https://security-tracker.debian.org/tracker/CVE-2017-8283"
-         ]
-       },
-       ...
-     ]
-   }
-   ```
+    {
+        "generated_at": "2021-03-09T11:40:28.154072066Z",
+        "artifact": {
+          "repository": "library/docker",
+          "digest": "sha256:7215e8e09ea282e517aa350fc5380c1773c117b1867316fb59076d901e252d15",
+          "mime_type": "application/vnd.docker.distribution.manifest.v2+json"
+        },
+        "scanner": {
+          "name": "Trivy",
+          "vendor": "Aqua Security",
+          "version": "v0.16.0"
+        },
+        "severity": "High",
+      "vulnerabilities": [
+          {
+              "id": "CVE-2020-1967",
+              "package": "libcrypto1.1",
+              "version": "1.1.1c-r0",
+              "fix_version": "1.1.1g-r0",
+              "severity": "High",
+              "description": "Server or client applications that call the SSL_check_chain() function during or after a TLS 1.3 handshake may crash due to a NULL pointer dereference as a result of incorrect handling of the \"signature_algorithms_cert\" TLS extension. The crash occurs if an invalid or unrecognised signature algorithm is received from the peer. This could be exploited by a malicious peer in a Denial of Service attack. OpenSSL version 1.1.1d, 1.1.1e, and 1.1.1f are affected by this issue. This issue did not affect OpenSSL versions prior to 1.1.1d. Fixed in OpenSSL 1.1.1g (Affected 1.1.1d-1.1.1f).",
+              "links": [
+                "https://avd.aquasec.com/nvd/cve-2020-1967"
+              ],
+              "cvss": {
+                "nvd": {
+                  "v2_vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
+                  "v3_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+                  "v2_score": 5,
+                  "v3_score": 7.5
+                },
+                "redhat": {
+                  "v3_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+                  "v3_score": 7.5
+                }
+              },
```

Comment:
The `cvss` property is also specific to Trivy. We introduced it in the past for some other integration, but in API v1.1 it's redundant. See `.vendor_attributes.CVSS`.